app/constants/date-format.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  DATE: 'DD/MM/YYYY HH:mm'
```

Comment:
Should the time format not be HH:MM?

Reply:
Unless I'm mistaken

Reply:
Can't do `MM` as `MM` is months.  `mm` is minutes.

Seconds were removed as agreed with that minutes would be detailed enough until we get feedback they really want to know the exact second something happened.

Reply:
Yep, sounds reasonable